kubernetes/src/main/java/io/kubernetes/client/openapi/apis/BatchV1Api.java

```diff
@@ -13,26 +13,21 @@
 
 package io.kubernetes.client.openapi.apis;
 
+import com.google.gson.reflect.TypeToken;
```

Comment:
This code is all generated, we don't edit it in this project.

Reply:
ok, where all api methods gets defined?

Reply:
Just trying to understand... You mean we can not change api methods for separate project aka adding BatchV1Api.createNamespacedJobFromCronJob, we can add util method for conversion, call it by passing cronJob object and pass result to BatchV1Api.createNamespacedJob to run job. Correct?

Reply:
Yes, that's correct. Basically, we don't edit things in the `kubernetes` directory (mostly, there is a `custom` dir in there...)

Reply:
Examples self generated as well? Wonder if I can add example class to show how this work.